packages/styles/scss/patterns/sub-patterns/content-section/_content-section.scss

```diff
@@ -0,0 +1,64 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '../../../globals/imports';
+
+@mixin themed-items {
+  color: $text-01;
+  background: $ui-background;
+}
+
+@mixin content-section {
+  .#{$prefix}--content-section {
+    @include carbon--make-row;
+
+    &__heading {
+      margin-bottom: $carbon--layout-03;
+      @include carbon--type-style('expressive-heading-04', true);
+
+      &__col {
+        @include carbon--make-col-ready;
+        @include carbon--make-col(4, 4);
```

Comment:
I think this just adds a 100%. Is that necessary for a block element that's already 100%?